### DISCUSSION TITLE
onclick on p-button should open image preview instead of clicking on image

### DISCUSSION THREAD

**Discussion by ekkalurusubhash** [primefaces/community#2437] [PrimeNG]
instead of click on image indicator to preview image, it should work when i am click on show button 
like below code .... it should work for image preview as well
`<div class="card flex justify-content-center">
    <p-button 
    icon="pi pi-external-link" 
    label="Show" 
    (onClick)="displayBasic = true" />
        <p-galleria
            [(value)]="images"
            [(visible)]="displayBasic"
            [responsiveOptions]="responsiveOptions"
            [containerStyle]="{ 'max-width': '850px' }"
            [numVisible]="7"
            [circular]="true"
            [fullScreen]="true"
            [showItemNavigators]="true"
            [showThumbnails]="false">
                <ng-template pTemplate="item" let-item>
                    <img 
                        [src]="item.itemImageSrc" 
                        style="width: 100%; display: block;" />
                </ng-template>
        </p-galleria>
</div>`

**Comment by EnricoMessall** [comment]
Do you mean image preview like https://primeng.org/image#preview or do you mean the fullscreen of p-galleria that you use? Showing p-galleria is working with the "fullScreen" property dynamic instead of visible. See component api docs:

visible: "Specifies the visibility of the mask on fullscreen mode."
fullScreen: "Whether to display the component on fullscreen."
